include/tacho_motor_class.h

```diff
@@ -194,9 +194,4 @@ extern void unregister_tacho_motor(struct tacho_motor_device *);
 
 extern struct class tacho_motor_class;
 
-inline struct tacho_motor_device *to_tacho_motor(struct device *dev)
```

Comment:
FWIW, there are lots of `static inline`s in other header files in the kernel - if we ever need to move this back. But, since it is not used anywhere else, moving it is fine.